corehq/apps/celery/__init__.py

```diff
@@ -18,3 +18,25 @@ def _init_celery_app():
     app.config_from_object('django.conf:settings', namespace='CELERY')
     app.autodiscover_tasks()
     app.set_default()
+
+
+class PeriodicTask(Task):
+
+    @classmethod
+    def on_bound(cls, app):
+        app.conf.beat_schedule[cls.name] = {
```

Comment:
Will this blow away a previous schedule if there was one for `cls.name`? Should it verify that there was no previous schedule, maybe add `assert cls.name not in app.conf.beat_schedule, cls.name` above this line?

Reply:
I tested it locally. It does not blow up. It just ignores any other task with the same name.
If there are tasks with the same names in different task files, I guess their `cls.name` would be different. So assert would not catch it. And if we define two tasks with the same name in the same task file then the stickler will complain about function redefinition. 
It would not harm to add assert but wondering if it would it be beneficial?